### PR TITLE
Move "winner" and "duration" texts in the summary

### PR DESCRIPTION
**Pull request by BrownHedgehog78**
"Fixes" this:
image

### CODE REVIEW

**Comment by savilli** [comment]
How does it look after the change?

**Comment by BrownHedgehog78** [comment]
> How does it look after the change?

image

**Comment by savilli** [comment]
How about you move "winner" to the left? I don't like free space there.

**Comment by BrownHedgehog78** [comment]
> How about you move "winner" to the left? I don't like free space there.

Like it was before?

**Comment by savilli** [comment]
> > How about you move "winner" to the left? I don't like free space there.
> 
> Like it was before? image

That looks good

**Comment by Panquesito7** [comment]
> > How does it look after the change?
> 
> image

For me, it looks better like this. The winner text being on the left leaves too much space on the right which makes it look bad, IMO.

**Comment by BrownHedgehog78** [comment]
uh, where should it be then?

File: mods/ctf/ctf_modebase/summary.lua [hunk 1]
```diff
@@ -232,7 +232,7 @@ function ctf_modebase.summary.show_gui_sorted(name, rankings, special_rankings,
 	if formdef.duration then
 		formspec.elements.duration = {
 			type = "label",
-			pos = {"center", 0.5},
+			pos = {14.5, 0.5},
```
Comment (LoneWolfHT) [inline mods/ctf/ctf_modebase/summary.lua:235]: I think you can use "right" here

**Comment by LoneWolfHT** [comment]
> > How does it look after the change?
> 
> image
> 

Is that middle element using `"center"`?

**Comment by BrownHedgehog78** [comment]
> > > How does it look after the change?
> > 
> > 
> > image
> 
> Is that middle element (Player x captured) using `"center"`?

Yes

File: mods/ctf/ctf_modebase/summary.lua (hunk 1 above)
Comment (BrownHedgehog78) [reply mods/ctf/ctf_modebase/summary.lua:235]: No, it crashes the server

**Comment by savilli** [comment]
Closed in favor of d80f2f1a00b51e43cd8c35d1298714b636e97d7a

**Pull request closed without merging**